code/datums/status_effects/rogue/debuff.dm

```diff
@@ -87,34 +87,6 @@
 	desc = "My BANE!"
 	icon_state = "hunger3"
 
-
-// PINTLEDESTRUCTION
-
-/datum/status_effect/debuff/pintledestruction
-	id = "pintlegone"
```

Comment:
why

Reply:
pintle also adds a stress event not just limpdick

Reply:
why as in why did i remove it or like "why did this code exist"

Reply:
why did you remove it outside of the limb dick it also gave a stress event

Reply:
i removed it cause it made sense to remove, aberra wants to gut and replace rather than keep it.

Reply:
but it doesn't replace the winebubble stress event in anyway just makes the winebubble worse